compiler/mx.compiler/suite.py

```diff
@@ -2047,7 +2047,7 @@
           "jdk.unsupported" # sun.misc.Unsafe
         ],
         "exports" : [
-          "* to com.oracle.graal.graal_enterprise,org.graalvm.nativeimage.pointsto,org.graalvm.nativeimage.builder,org.graalvm.nativeimage.llvm,com.oracle.svm.svm_enterprise,com.oracle.svm_enterprise.ml_dataset,org.graalvm.nativeimage.base",
+          "* to com.oracle.graal.graal_enterprise,org.graalvm.nativeimage.pointsto,org.graalvm.nativeimage.pointsto.standalone,org.graalvm.nativeimage.builder,org.graalvm.nativeimage.llvm,com.oracle.svm.svm_enterprise,com.oracle.svm_enterprise.ml_dataset,org.graalvm.nativeimage.base",
```

Comment:
Can we avoid adding this exports to `pointsto.standalone`? It creates some problems with the module system in our CI. Could you instead use the `ModuleSuport` API like you do for other cases in `PointsToAnalyzer`?

Reply:
Fixed by removing the export